Userland/Libraries/LibCore/System.cpp

```diff
@@ -1655,4 +1655,39 @@ ErrorOr<void> posix_fallocate(int fd, off_t offset, off_t length)
 }
 #endif
 
+// This constant is copied from LibFileSystem. We cannot use or even include it directly,
+// because that would cause a dependency of LibCore on LibFileSystem, effectly rendering
+// the distinction between these libraries moot.
```

Comment:
I _think_ the original reason for the distinction was that we didn't really want to create yet another sub-namespace inside `LibCore`, or at least some people were vehemently against it. I'd have to look it up in the original PR again.

Reply:
Looks like it was changed away from a sub-namespace only close to the end: https://github.com/SerenityOS/serenity/pull/17789#pullrequestreview-1350262636

Anyway, I agree that I don't want to re-open that can of worms. That's why I suggest to move this one function back to LibCore, and keep the rest in LibFileSystem.

We could also "officially" move DEFAULT_PATH and DEFAULT_PATH_SV to LibCore, but that raises the question which namespace to use. Do you have an idea where it could go, or is this okay for now?